### ISSUE TITLE
Error: query was not given

### ISSUE THREAD

**Issue by SirGryphin** [dipu-bd/lightnovel-crawler#691]
**Describe the bug**
Get this error when running lncrawl.

` ! Error: Novel page url or query was not given`

![Screenshot 2020-12-13 183553](https://user-images.githubusercontent.com/36343615/102020603-17375380-3d72-11eb-9081-627dede9ed2b.png)

**Let us know**
- Novel URL: None
- App Version: 2.24.0
- Package: pip
- OS:  Windows 10 Pro 20H2 19042.630

**Comment by dipu-bd** [comment]
I replaced `PyInquirer` with `questionary` for new CLI. It might be the cause

**Comment by amritoo** [comment]
```
PS C:\Users\Amrito> lncrawl -lll
================================================================================
                            Lightnovel Crawler #2.24.0
                  https://github.com/dipu-bd/lightnovel-crawler
--------------------------------------------------------------------------------
                          << LOG LEVEL: DEBUG
--------------------------------------------------------------------------------
2020-12-14 21:25:34,850 [DEBUG] (urllib3.connectionpool)
Starting new HTTP connection (1): bit.ly:80
Failed to check for update

-> Press  Ctrl + C  to exit

2020-12-14 21:25:40,329 [INFO] (lncrawl.utils.racovimge)
Jinja2 is required for cover generation:
    pip install Jinja2
2020-12-14 21:25:40,334 [INFO] (lncrawl.core.downloader)
CairoSVG was not found.Install it to generate random cover image:
    pip install cairosvg
2020-12-14 21:25:40,342 [INFO] (lncrawl.core.app)
Initialized App
2020-12-14 21:25:40,376 [DEBUG] (asyncio)
Using proactor: IocpProactor
Traceback (most recent call last):
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\lncrawl\bots\console\get_crawler.py", line 32, in get_novel_url
    answer = prompt([
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\questionary\prompt.py", line 28, in prompt
    return unsafe_prompt(questions, answers, patch_stdout, true_color, **kwargs)
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\questionary\prompt.py", line 118, in unsafe_prompt
    answer = question.unsafe_ask(patch_stdout)
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\questionary\question.py", line 64, in unsafe_ask
    return self.application.run()
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\prompt_toolkit\application\application.py", line 816, in run
    return loop.run_until_complete(
  File "c:\users\amrito\scoop\apps\python\current\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\prompt_toolkit\application\application.py", line 783, in run_async
    return await _run_async2()
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\prompt_toolkit\application\application.py", line 765, in _run_async2
    result = await _run_async()
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\prompt_toolkit\application\application.py", line 701, in _run_async
    self._redraw()
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\prompt_toolkit\application\application.py", line 536, in _redraw
    self.context.run(run_in_context)
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\prompt_toolkit\application\application.py", line 522, in run_in_context
    self.renderer.render(self, self.layout)
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\prompt_toolkit\renderer.py", line 698, in render
    output.flush()
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\prompt_toolkit\output\windows10.py", line 53, in flush
    self.vt100_output.flush()
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\prompt_toolkit\output\vt100.py", line 666, in flush
    out.write(data.encode(self.stdout.encoding or "utf-8", "replace"))
  File "C:\Users\Amrito\AppData\Roaming\Python\Python38\site-packages\colorama\ansitowin32.py", line 41, in write
    self.__convertor.write(text)
  File "C:\Users\Amrito\AppData\Roaming\Python\Python38\site-packages\colorama\ansitowin32.py", line 162, in write
    self.write_and_convert(text)
  File "C:\Users\Amrito\AppData\Roaming\Python\Python38\site-packages\colorama\ansitowin32.py", line 184, in write_and_convert
    text = self.convert_osc(text)
  File "C:\Users\Amrito\AppData\Roaming\Python\Python38\site-packages\colorama\ansitowin32.py", line 246, in convert_osc    for match in self.ANSI_OSC_RE.finditer(text):
TypeError: cannot use a string pattern on a bytes-like object

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\amrito\scoop\apps\python\current\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\amrito\scoop\apps\python\current\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\Amrito\scoop\apps\python\current\Scripts\lncrawl.exe\__main__.py", line 7, in <module>
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\lncrawl\__init__.py", line 13, in main
    start_app()
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\lncrawl\core\__init__.py", line 79, in start_app
    raise err
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\lncrawl\core\__init__.py", line 76, in start_app
    run_bot(bot)
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\lncrawl\bots\__init__.py", line 17, in run_bot
    ConsoleBot().start()
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\lncrawl\bots\console\start.py", line 28, in start
    self.app.user_input = self.get_novel_url()
  File "c:\users\amrito\scoop\apps\python\current\lib\site-packages\lncrawl\bots\console\get_crawler.py", line 42, in get_novel_url
    raise Exception('Novel page url or query was not given')
Exception: Novel page url or query was not given
```

**Comment by dipu-bd** [comment]
fixed by 2.24.1
